<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-JGU2O17" failures="2" tests="2" name="Test(failed)" time="32.032" errors="0" timestamp="2025-06-23T22:58:14 IST">
  <testcase classname="testCases.TC_001_AccountRegistrationTest" name="verify_account_registration" time="14.094">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at testCases.TC_001_AccountRegistrationTest.verify_account_registration(TC_001_AccountRegistrationTest.java:64)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- verify_account_registration -->
  <testcase classname="testCases.TC003_Login_DDT" name="loginddt" time="0.0">
    <failure type="org.testng.TestNGException" message="
Method public void testCases.TC003_Login_DDT.loginddt(java.lang.String,java.lang.String,java.lang.String) requires a @DataProvider named : Logindata in class Utilities.Dataprovider">
      <![CDATA[org.testng.TestNGException: 
Method public void testCases.TC003_Login_DDT.loginddt(java.lang.String,java.lang.String,java.lang.String) requires a @DataProvider named : Logindata in class Utilities.Dataprovider
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- loginddt -->
</testsuite> <!-- Test(failed) -->
